helm-chart/renku/templates/network-policies.yaml

```diff
@@ -1,3 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-all-ingress
+  labels:
+    app: {{ template "renku.name" . }}
+    chart: {{ template "renku.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  podSelector: {}
```

Comment:
Will this policy deny ingress to **all** pods in the cluster? e.g. also pods that have nothing to do with Renku?...
Maybe I am wrong, but it reads like that...
If I am not wrong then we should make this policy be "deny all ingress to Renku pods".

Reply:
This denies ingress to ALL pods in the namespace where the helm chart is installed. Not to all pods in the cluster.

Reply:
I did not want to call it "renku pods" because you may have other things in the namespace other than renku pods.